tests/e2e/testdata/scenarios/direct/llm_immutable_field/labels_value_type/_object01.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       bounds:
       - 1.5
       - 4.5
-  description: an updated description
+  description: a description
```

Comment:
👀 would we have expected this to not get updated?

Reply:
Ah yeah, the `description` field should not be updated: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/testdata/scenarios/direct/llm_immutable_field/labels_value_type/script.yaml#L55

Reply:
Oh I see, I had gotten confused!